getting-started/windows-calico/requirements.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Requirements 
+description: Review requirements for the standard install for Calico for Windows.
+canonical_url: '/getting-started/windows-calico/requirements'
+---
+
+### About {{site.prodnameWindows}}
+
+Because the Kubernetes and {{site.prodname}} control components do not run on Windows yet, a hybrid Linux/Windows cluster is required. {{site.prodnameWindows}} standard installation is distributed as a **.zip archive**. 
+
+### What's supported in this release
+
+✓ Install: Manifest install for Kubernetes 2 clusters
```

Comment:
Similar to my comment above re: K2, is this what we're using @bmckercher123 ?

Reply:
As above.